integration/models/application.ts

```diff
@@ -70,7 +71,16 @@ export const application = (config: ApplicationConfig, appDirPath: string, appDi
     },
     build: async () => {
       const log = logger.child({ prefix: 'build' }).info;
-      await run(scripts.build, { cwd: appDirPath, log });
+      await run(scripts.build, {
+        cwd: appDirPath,
+        log: (msg: string) => {
+          buildOutput += `\n${msg}`;
+          log(msg);
+        },
```

Comment:
Captures and makes the build output available by way of `app.buildOutput`. This allows us to assert on the output.